tests/test_download_cli_integration.py

```
@@ -0,0 +1,765 @@
+
+def test_validate_integration_missing_components():
+    """validate_integration should return False when components are missing."""
+    integration = DownloadCLIIntegration()
+    integration.orchestrator = None
+
+    result = integration.validate_integration()
+
+    assert result is False
+
+
+def test_validate_integration_fetch_failure(mocker):
```

Ruff (0.14.8)

Unused function argument: `mocker`

(ARG001)